plugins/copr.py

```diff
@@ -482,9 +483,19 @@ def _download_repo(self, project_name, repo_filename):
                     import urllib
                     res = urllib.urlopen(self.copr_url + "/coprs/" + project_name)
                     status_code = res.getcode()
-                if str(status_code) != '404':
-                    raise dnf.exceptions.Error(_("This repository does not have"
-                                                 " any builds yet so you cannot enable it now."))
+                if status_code != 404:
+                    error_msg = "It wasn't possible to enable this project.\n"
+                    try:
+                        if PY3:
+                            response = urllib.request.urlopen(self.copr_url + api_path)
+                        else:
+                            response = urllib.urlopen(self.copr_url + api_path)
+                    except urllib.error.HTTPError as e:
+                        msg = e.headers.get("Error-Message")
+                        if msg:
+                            msg = base64.b64decode(msg).decode('utf-8')
+                            error_msg = ''.join([error_msg, msg])
+                    raise dnf.exceptions.Error(_(error_msg))
```

Comment:
I'm not sure translation via `_()` is doable here, as the message is server-side generated thingy.  Therefore, eventually, I'd prefer if we could  send the list of available chroots, and fully construct the message here in copr.py plugin ...

Reply:
To make the message translatable, you probably need to do something like `_("Available chroots: {}").format()`.  The string for translation needs to be a constant.